lib/web_console/templates/console.js.erb

```diff
@@ -864,11 +864,23 @@ REPLConsole.prototype.scrollToBottom = function() {
   this.outer.scrollTop = this.outer.scrollHeight;
 };
 
-// Change the binding of the console
+/**
+ * Change the binding of the console
+ * Also updates a div to show that it has been changed.
+ */
 REPLConsole.prototype.switchBindingTo = function(frameId, callback) {
   var url = this.getSessionUrl('trace');
   var params = "frame_id=" + encodeURIComponent(frameId);
-  postRequest(url, params, callback);
+  var notification = document.createElement('div');
+  var _this = this;
+
+  notification.className = "console-notification";
```

Comment:
```js
var notification = document.createElement('div');
notification.className = "console-notification";

var _this = this;
postRequest(url, params, function() {
```

Can we keep the flow like this here? The operations are "grouped" this way.

Reply:
sure!